ext/http/00_serve.ts

```diff
@@ -370,7 +370,25 @@ class InnerRequest {
       return null;
     }
     this.#streamRid = op_http_read_request_body(this.#external);
-    this.#body = new InnerBody(readableStreamForRid(this.#streamRid, false));
+    this.#body = new InnerBody(
+      readableStreamForRid(
+        this.#streamRid,
+        false,
+        undefined,
+        (controller, error) => {
+          if (ObjectPrototypeIsPrototypeOf(BadResourcePrototype, error)) {
+            // TODO(kt3k): We would like to pass `error` as `cause` when BadResource supports it.
```

Comment:
Hm, shouldn't this be doable already?

Reply:
Doesn't look so as they are implemented like:

```js
  class BadResource extends Error {
    constructor(msg) {
      super(msg);
      this.name = "BadResource";
    }
  }
```
https://github.com/denoland/deno_core/blob/8d7bfea48ebfcaee989f8d72b243fb09302014f9/core/01_core.js#L302-L305

Reply:
Created an issue in deno_core https://github.com/denoland/deno_core/issues/1031